application/forms/Config/General/LoggingConfigForm.php

```diff
@@ -38,12 +54,7 @@ public function createElements(array $formData)
                 'autosubmit'    => true,
                 'label'         => $this->translate('Logging Type'),
                 'description'   => $this->translate('The type of logging to utilize.'),
-                'multiOptions'  => array(
-                    'syslog'    => 'Syslog',
-                    'php'       => $this->translate('Webserver Log', 'app.config.logging.type'),
-                    'file'      => $this->translate('File', 'app.config.logging.type'),
-                    'none'      => $this->translate('None', 'app.config.logging.type')
-                )
+                'multiOptions'  => $types
```

Comment:
Please reduce this complexity monster to this:

```
'value' => Platform::isDocker() ? 'php' : 'syslog'
```

Nothing else than this should be required.

Reply:
😅